bigbluebutton/greenlight
# add(feature): SuperAdmin [2]

- Add tenant#index
- Add pagination
- Add search feature

Admin should now be able to see all the tenants, search the tenants, add a new tenant.

**How To Test:**

- Create over 10 tenants
- Check that all the tenants are listed
- Test the search function
- Test the pagination

![image](https://user-images.githubusercontent.com/43917914/224133017-ee8aebf7-1f49-4fa2-9f25-6a441d31fd5b.png)
